src/App.tsx

```diff
@@ -9,39 +11,23 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
-import { Post } from './types/Post';
+import { useAppDispatch, useAppSelector } from './app/hooks';
```

Comment:
Ensure that the hooks 'useAppDispatch' and 'useAppSelector' are correctly defined and exported in './app/hooks'.